argilla/mkdocs.yml

```diff
@@ -138,7 +138,8 @@ nav:
       - Manage workspaces: how_to_guides/workspace.md
       - Manage and create datasets: how_to_guides/dataset.md
       - Add, update, and delete records: how_to_guides/record.md
-      - Query, filter, and export records: how_to_guides/query_export.md
+      - Query and filter records: how_to_guides/query.md
+      - Importing and exporting datasets: how_to_guides/export.md
```

Comment:
@burtenshaw we should change the file (and URL) name, instead of `export` we could ´import-export´ I think it'd help SEO